package.json

```diff
@@ -23,6 +24,13 @@
   "author": "Daniel Lytkin <dan.lytkin@gmail.com>",
   "license": "MIT",
   "devDependencies": {
-    "typescript": "^2.2.1"
+    "@babel/core": "^7.24.7",
+    "@babel/preset-env": "^7.24.7",
+    "@babel/preset-typescript": "^7.24.7",
+    "@types/jest": "^29.5.12",
+    "babel-jest": "^29.7.0",
```

Comment:
ℹ️ FYI, `babel` is used just for allowing `jest` to run typescript tests. The `index.js` and `index.d.ts` files are still generated purely through `tsc`.